website/src/content/main-concepts/user-permissions.mdx

```diff
@@ -2,10 +2,212 @@
 title: User Permissions
 ---
 
+The Merchant Center allows configuring permissions based on a user’s team. Teams belong to an organization and users can be member of multiple teams.
+
+See [User Permission Concepts](https://docs.commercetools.com/merchant-center/user-permissions.html) for more information about the relationships between users, teams, and organizations.
+
+# Types of permissions
+
+The Merchant Center exposes different types of user permissions that are bound to a project and can be used within a Custom Application code. For example, to restrict access to a view, to prevent updating or deleting certain resources, etc.
+
+## General permissions
+
+General permissions are bound to a resource and grant either **view** or **manage** access.
+
 <Info>
 
-See [User Permission Concepts](https://docs.commercetools.com/merchant-center/user-permissions.html).
+A manage access always implies view access as well.
 
 </Info>
```

Comment:
I assume that everybody has view access. Right?

Suggestion:
General permissions are bound to a resource and always imply view access. Additionally to **view** access, the **manage** access allows to perform ... *fill in what the manage access differentiates from view access*

Reply:
No. View access is also something which needs to be explicitely configured.